book/src/pong-tutorial/pong-tutorial-03.md

```diff
@@ -340,14 +350,12 @@ Our run function should now look something like this:
 
 ## Automatic set up of resources by system.
 
-You might remember, that we had troubles because
-amethyst required to set up storage for `Paddle` before
-we could use it.
+You might remember, that we had troubles because Amethyst required
```

Comment:
Not on you, but this sentence is odd. Should it be: "Amethyst requires _us_ to ..."?